chrome/android/java/src/org/chromium/chrome/browser/preferences/Preferences.java

```diff
@@ -189,6 +196,10 @@ protected void onCreate(Bundle savedInstanceState) {
     protected void onDestroy() {
         super.onDestroy();
 
+        if (AdblockMode.sManageSubscriptions) {
```

Comment:
> What about checking whether subscriptionsManager is null? It can be the second condition in the if.

It can't be `null`, it's created when in `onCreate()` the condition is checked. There is no other places to release it before. `onDestroy` happens only once